proposals/dark_mode_extension_icons.md

```diff
@@ -0,0 +1,158 @@
+# Dark Mode Extension Icons
+
+**Summary**
+
+Feature to enable developers to enhance extension icon visibility in dark mode.
+
+**Document Metadata**
+
+**Author:** &lt;solomonkinard>
+
+**Sponsoring Browser:** Chromium.
+
+**Contributors:** &lt;oliverdunk>
+
+**Created:** 2024-04-05
+
+**Related Issues:** &lt;https://crbug.com/893175>
+
+## Motivation
+
+### Objective
+
+Extension developers will be able to supply and define a set of icons to be used
+in the event that browser system settings are set to dark mode. The set of dark
+mode icons can and will be made automatically visible once the system setting is
+in dark mode, thereby enhancing the overall browser experience.
+
+#### Use Cases
+
+1. Improved icon visibility in the extension toolbar.
+2. Improved icon visibility on management the management and shortcuts pages.
```

Comment:
Suggest adding following use cases:
- Improved icon visibility on context menu (Chrome and Safari reply on default icons, only Firefox can specify icons)
- Improved icon visibility on side panel (Chrome replies on default icons, Firefox  can specify icons)

Reply:
Addressed in the patch response.